### PR TITLE
Parse koji events correctly

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Fixes: #1350

---

N/A

### CODE REVIEW

**Review by jpopelka — APPROVED** [review]
I'm glad you solved it. Good catch.

File: tests/unit/test_events.py [hunk 1]
```diff
@@ -177,70 +177,6 @@ def distgit_commit(self):
...
-        ) as outfile:
-            return json.load(outfile)
-
-    @pytest.fixture()
-    def koji_build_completed_epel8(self):
-        with open(
-            DATA_DIR / "fedmsg" / "koji_build_completed_epel8.json", "r"
-        ) as outfile:
-            return json.load(outfile)
-
```
Comment (TomasTomecek) [inline tests/unit/test_events.py:243]: would it make sense to drop unused files from the repo?

**Review by TomasTomecek — APPROVED** [review]
wow, nicely spotted! lovely to see as well to have it covered with tests

File: tests/unit/test_events.py (hunk 1 above)
Comment (lachmanfrantisek) [reply tests/unit/test_events.py:243]: The files are still in use. We just had the fixtures defined multiple times..;)

**Pull request merged**